### PR TITLE
Actions for iTerm2.

### PR DESCRIPTION
**Pull request by melwin**
Actions for iTerm2 (https://www.iterm2.com/).

### CODE REVIEW

**Comment by melwin** [comment]
@larsblumberg Hey Lars, you guys interested in more PRs?

File: Application Actions/iTerm/1.0/default.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "RotateLeft"  : "Scroll Down",
+  "RotateRight" : "Scroll Up",
+  "SwipeUp"     : "Previous Pane",
+  "SwipeDown"   : "Next Pane",
+  "SwipeLeft"   : "Previous Tab",
+  "SwipeRight"  : "Next Tab"
```
Comment (larsblumberg) [inline Application Actions/iTerm/1.0/default.json]: To be consistent with existing applications (e.g. Safari, Chrome, Firefox), can we rename those to

```
Select Previous Pane
Select Next Pane
Select Previous Tab
Select Next Tab
```

Thanks!

**Comment by larsblumberg** [comment]
@melwin Yes, more PRs much appreciated!

**Comment by melwin** [comment]
@larsblumberg How's that? Note I used `keystroke` here as I couldn't determine that key codes for [ and ] are consistent across languages, as per previous discussion.

**Comment by larsblumberg** [comment]
@melwin – I'd assume the keyboard shortcuts to be the same for iTerm across different languages. So it's a good thing to use `keystroke` as you do here. It should work across different keyboard layouts. Thanks for this iTerm2 integration – I'll merge it.

**Pull request merged**